### DISCUSSION TITLE
Metrics Errors

### DISCUSSION THREAD

**Discussion by jcouzy** [charlie-haley/omada_exporter#54] [General]
Hi, when I run the Omaha_exporter in  docker container is get errors when trying to access the metrics on http://hostname:9202.
I run the container like this:
docker run -d --name=omada-exporter -p 9202:9202 -e OMADA_HOST='https://omada.infra.local' -e OMADA_USER='prometheus' -e OMADA_PASS='*************' -e OMADA_INSECURE=true chhaley/omada_exporter

When I try to access the metrics, I receive these errors:

An error has occurred while serving metrics:

144 error(s) occurred:
* collected metric "omada_port_power_watts" { label:<name:"client" value:"" > label:<name:"profile" value:"All" > label:<name:"site" value:"Default" > label:<name:"site_id" value:"Default" > label:<name:"switch_id" value:"60f03f07447e14066d93e7f9" > label:<name:"switch_mac" value:"00-31-92-76-44-6C" > label:<name:"switch_port" value:"1" > label:<name:"vendor" value:"" > label:<name:"vlan_id" value:"1" > gauge:<value:5.2 > } was collected before with the same name and label values 

And on and on.

What am I doing wrong?

**Comment by charlie-haley** [comment]
I recently updated the repo to use collectors, it's possible there's a bug here, I'll create an issue and have a look.

What version of the omada controller are you running?

**Reply by jcouzy** [reply]
I am running version 5.6.4 on an OC200.
